glommio/src/task/raw.rs

```diff
@@ -221,6 +227,9 @@ where
             dbg_context!(ptr, "foreign", {
                 let notifier = raw.notifier();
                 notifier.queue_waker(Waker::from_raw(Self::clone_waker(ptr)));
+                if (*raw.header).latency_matters {
```

Comment:
If we use a separate eventfd, we can then write `if Some(fd) = latency_matters { write(fd) }`